src/pages/Career/redux/thunk.js

```diff
@@ -0,0 +1,14 @@
+import {  fetchJobsSuccess, fetchJobsFailure } from './slice';
```

Comment:
You have commented on these actions in the slice. Please recheck the slice. js.

Reply:
updated in latest commit
this was older one